pkg/utils/kubeutils/portforward/cli_portforwarder.go

```diff
@@ -61,6 +61,8 @@ func (c *cliPortForwarder) startOnce(ctx context.Context) error {
 		cmdCtx,
 		"kubectl",
 		"port-forward",
+		"--context",
+		c.properties.kubeContext,
```

Comment:
Does this not need an empty check?
Ie if kubecontext is empty then --context with an empty arg mgiht not work as expected in this pass to cmd functionality?
Im too lazy to check but would love that assurance!

Reply:
Not specifying `--kube-context` in the glooctl command functions as expected on my local kube cluster. Does that address your concern, or is it something deeper?